docs/Setup.md

```diff
@@ -16,6 +16,14 @@ daemon InfraP4d of the IPDK networking recipe to be runnning in the background.
 Once InfraP4d is running, Kubernetes can load its P4 pipeline and offload
 various functionalities on it (i.e. on the P4 data plane).
 
+On the Intel IPU, k8s-infra olload can run in 2 different modes details of which
```

Comment:
k8s-infra offload. Please correct the spell error.

Reply:
done